test/lpdf.cc

```diff
@@ -1,13 +1,13 @@
 #include <gtest/gtest.h>
 
-#include <Eigen/Dense>
 #include <stan/math/prim/fun.hpp>  // lgamma, lmgamma
 #include <stan/math/prim/prob.hpp>
+#include <stan/math/rev.hpp>
 
 #include "algorithm_state.pb.h"
 #include "src/hierarchies/lin_reg_uni_hierarchy.h"
 #include "src/hierarchies/nnig_hierarchy.h"
-#include "src/hierarchies/nnw_hierarchy.h"
+// #include "src/hierarchies/nnw_hierarchy.h"
```

Comment:
+1 for commented lines

Reply:
Do we have the correct lpdf::nnw test now?

Reply:
I checked it out: It requires a lot of work to fix the code to compile test_bayesmix. I would simply remove it, or at least preserve it away from the release.